cob_hardware_config/cob4-2/config/flexisoft.yaml

```diff
@@ -1,9 +1,9 @@
 host: 10.4.2.99
 port: "9100" # alternative syntax: !!str 9100
 odometry_timeout: 1.0
-threshold_linear_slow: 0.1
-threshold_linear_fast: 0.4
-threshold_angular_fast: 0.5
+threshold_linear_slow: 1.1
```

Comment:
we currently need to shrink the safety fields down to a minimum so that they are within the navigation's footprint. But this setup is not good for running 24/7 tests within MSH.

In order to use the desired safety fields which are selected based on the current velocity of the robot we need to find a way that the navigation can deal with changing footprints. --> @ipa-srd: do you have a solution ready?